simnode/generator.py

```diff
@@ -156,44 +156,47 @@ def add_obstacles_to_scenario(self, scenario: Scenario) -> None:
                 mesh = ProceduralBump(pos, rot, obstacle.width, obstacle.length, height, obstacle.upper_length,
                                       obstacle.upper_width)
             elif obstacle_type is Stopsign:
+                rot = (0, 0, -(obstacle.z_rot - 90))
```

Comment:
This assumes that the environment that DriveBuild uses is flat.
Just in theory: At some day in the future it may allow hills or pits. As soon as this is possible one wants to define xRot and yRot as well.
Another nailcare: "-(obstacle.z_rot - 90)" is equivalent to "90 - obstacle.z_rot" and I think it is easier to read.